src/controllers/oracle.ts

```diff
@@ -58,7 +58,7 @@ export default class OralceController {
         const response = await axios.get(`${this.url}/${ticker}`);
         if (response.status !== 200) throw new Error(response.data);
         const data: BitfinexResponse = response.data;
-        timestampToUse = data.timestamp;
+        timestampToUse = data.timestamp.split(".")[0];
```

Comment:
Math.trunc

Reply:
which is done later for attestation, we could say we should return trucated also in response